ray-operator/config/samples/ray_v1alpha1_rayjob.batch-inference.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: ray.io/v1alpha1
+kind: RayJob
+metadata:
+  name: rayjob-sample
+spec:
+  entrypoint: python /home/ray/samples/sample_code.py
+  # shutdownAfterJobFinishes specifies whether the RayCluster should be deleted after the RayJob finishes. Default is false.
```

Comment:
I suggest eliminating extraneous comments to minimize distractions. If these fields are not used / mentioned in the doc, it would be better to remove it.